source/getting-started/index.markdown

```diff
@@ -13,7 +13,7 @@ The goal of this getting started guide is to install Hass.io on a Raspberry Pi 3
 
 Follow this guide if you want to easily get started with Home Assistant, or if you have no or little Linux experience. For advanced users, check our [alternative installation methods](/docs/installation/).
 
-### {% linkable_title Hardware requirements %}
+### {% linkable_title Hardware requirements %
```

Comment:
Missing closing }

Reply:
Duh! That was a post-writeup test to get my Emacs to highlight possible syntax issues in markdown files, and somehow snuck in to the actual commit :( Force-pushed a new one with that removed.